sea-orm-cli/src/cli.rs

```diff
@@ -156,5 +157,29 @@ pub enum GenerateSubcommands {
             help = "Automatically derive serde Serialize / Deserialize traits for the entity (none, serialize, deserialize, both)"
         )]
         with_serde: String,
+
+        #[clap(
+            arg_enum,
+            value_parser,
+            long,
+            default_value = "chrono",
+            help = "The datetime crate to use for generating entities."
+        )]
+        date_time_crate: DateTimeCrate,
     },
 }
+
+#[derive(ArgEnum, Copy, Clone, Debug, PartialEq)]
+pub enum DateTimeCrate {
+    Chrono,
+    Time,
+}
+
+impl From<DateTimeCrate> for CodegenDateTimeCrate {
+    fn from(date_time_crate: DateTimeCrate) -> CodegenDateTimeCrate {
+        match date_time_crate {
+            DateTimeCrate::Chrono => CodegenDateTimeCrate::Chrono,
+            DateTimeCrate::Time => CodegenDateTimeCrate::Time,
+        }
+    }
+}
```

Comment:
Thanks! @nahuakang @ikrivosheev I think it's okay to re-export `sea-orm-codegen::DateTimeCrate` and use it on `sea-orm-cli` instead of defining it on each crate with conversion method.

Reply:
Oh... sorry please forget it. I think we do need a boundary between two crate. Hance, we should keep it as-is.